openfisca_core/entities.py

```diff
@@ -89,6 +194,22 @@ def filled_array(self, value, dtype = None):
             warnings.simplefilter("ignore")
             return np.full(self.count, value, dtype)
 
+    def get_holder(self, variable_name, init = True):
```

Comment:
Good point. Do you have an alternative suggestion ?

`get_or_build_holder` ?
`get_or_init_holder` ?